command/output.go

```diff
@@ -26,15 +27,19 @@ func (c *OutputCommand) Run(args []string) int {
 	}
 
 	args = cmdFlags.Args()
-	if len(args) != 1 || args[0] == "" {
+	var allOutputs bool = false
+	var name string
+	if len(args) > 1 {
 		c.Ui.Error(
 			"The output command expects exactly one argument with the name\n" +
-				"of an output variable.\n")
+				"of an output variable or no arguments to show all outputs.\n")
 		cmdFlags.Usage()
 		return 1
+	} else if len(args) == 0 {
```

Comment:
`else if` is uncommon in go - you can just lean on the `return` above and pop back up to the top level